samples/net/google_iot_mqtt/src/private_info/.gitignore

```diff
@@ -0,0 +1,4 @@
+*-cert.pem
```

Comment:
These are not needed if you put the pem/der files to version control and generate the .inc file automatically from them.

Reply:
This would be a security violation as you'd be checking in a private key that is unique to a user.